packages/react-native-sdk/src/components/Participant/ParticipantView/ParticipantVideoFallback.tsx

```diff
@@ -18,14 +18,35 @@ export type ParticipantVideoFallbackProps = Pick<
 export const ParticipantVideoFallback = ({
   participant,
 }: ParticipantVideoFallbackProps) => {
+  const {
+    theme: { colors, typefaces, participantVideoFallback },
+  } = useTheme();
   const { name, image, userId } = participant;
   const participantLabel = name ?? userId;
 
   // Display the Participant name/user id if the image isn't present.
   return (
-    <View style={styles.container}>
+    <View
+      style={[
+        styles.container,
+        {
+          backgroundColor: colors.static_grey,
+        },
+        participantVideoFallback.container,
+      ]}
+    >
       {!image ? (
-        <Text style={styles.label}>{participantLabel}</Text>
+        <Text
+          style={[
+            {
+              color: colors.static_white,
+            },
+            typefaces.bodyBold,
+            participantVideoFallback.label,
+          ]}
+        >
+          {participantLabel}
+        </Text>
       ) : (
         <Avatar participant={participant} />
```

Comment:
As a user, how can I add a theme to customize the Avatar here inside ParticipantFallback?

Reply:
hmm we faced this issue with chat sdk as well. I guess we can introduce some styles and forward them to underlying Avatar component. But we should only do this if its really needed. Since such styles can get hard to maintain soon.